processor/src/main/java/org/mapstruct/ap/internal/model/source/selector/XmlElementDeclSelector.java

```diff
@@ -112,4 +119,38 @@ else if ( nameMatches.size() > 0 ) {
             return methods;
         }
     }
+
+    private Pair getMatch(SourceMethod sourceMappingMethod, final String targetPropertyName) {
```

Comment:
why is `targetPropertyName` final and `sourceMappingMethod` not? Although I agree, its better to use the final keyword to indicate a variable is not modified (and its slightly better performing as well I once understood), I don't think we do that in other places of the code. So perhaps we should stick to the conventions and remove the final keywords.